start_cerberus.py

```diff
@@ -193,7 +99,7 @@ def main(cfg):
 
             # Monitor nodes status
             if watch_nodes == "True":
```

Comment:
Not a big deal but the remainder seems like it could be done with a loop of some point since its effectively the same code over and over w/ different if trues.

Reply:
Right, https://github.com/openshift-scale/cerberus/pull/18 will fix it do the checks in a loop like you mentioned.